kubernetes_platform/python/kfp/kubernetes/toleration.py

```diff
@@ -32,6 +32,7 @@ def add_toleration(
     value: Optional[str] = None,
     effect: Optional[Literal["NoExecute", "NoSchedule", "PreferNoSchedule"]] = None,
     toleration_seconds: Optional[int] = None,
+    toleration_json: Optional[Union[pipeline_channel.PipelineParameterChannel, dict]] = None,
```

Comment:
Optional: I think a new function name would be better than adding that additional keyword argument.